### DISCUSSION TITLE
Does Flashlist support RN v0.69.5 ?

### DISCUSSION THREAD

**Discussion by Daniyaalbeg** [Shopify/flash-list#580] [Q&A] [answered]
Ive upgraded my project to 0.69 and am trying to figure out what is causing my build errors. The :app:compileDebugJavaWithJavac step seems to always fail at the step when compiling flashlist citing   

```
Runtime JAR files in the classpath should have the same version. These files were found in the
...FILES HERE...
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
```

It builds on iOS, so I think its a gradle issue but not sure what to do. Tried tweaking gradle versions but no luck. 

Another part of the error:

`Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.`

Any help would be appreciated.

**Comment by fortmarek** [comment]
Hey @Daniyaalbeg!

Are you still seeing this issue? Yes, we do support 0.69.5, maybe you could create a reproducible sample and file an issue for this?

**Comment by Daniyaalbeg** [comment, marked as answer]
Hi, I managed to fix the error. It was not related to flash list.
